packages/components/src/popover/index.js

```diff
@@ -181,29 +205,6 @@ const Popover = (
 		? positionToPlacement( position )
```

Comment:
Good question — I don't think that `floating-ui` supports an equivalent position.

The plan is to:
 - convert all usages of `Popover` to use `placement` instead of `position`
 - mark `position` as deprecated / non-supported 
 - therefore, avoiding spending time on fixing it unless more reports are filed 😅